Gemfile

```diff
@@ -41,44 +40,45 @@ gem "acts_as_tree",                   "~>2.1.0"  # acts_as_tree needs to be requ
 # See miq_expression_spec Date/Time Support examples.
 # https://github.com/jeremyevans/ruby-american_date
 gem "american_date"
```

Comment:
@chessbyte I'm pretty sure it is. From my past experience on another project, american_date was required as long as dates in the format mm/dd/yyy are still being used. @jrafanie can you confirm?